timelock-api/build.gradle

```diff
@@ -10,6 +10,8 @@ conjure {
 }
 
 dependencies {
+    compile project(":leader-election-impl")
```

Comment:
Let's try to avoid dependencies on `leader-election-impl` in `timelock-api`. For example, this means internal backup service will be able to create a Paxos leader election service impl, which feels off.